packages/components/header/header.js

```diff
@@ -224,7 +224,12 @@ class Header {
 
     // Move items based on available width
     items.forEach((item) => {
-      if (item.right <= this.navigationList.offsetWidth) {
+      const maxRight =
```

Comment:
I think this would be a bit clearer if you handled the menu bar width on the other side of the comparison, e.g. outside the scope of the forEach you can have

```js
let availableSpace = this.navigationList.offsetWidth
if (items.length !== itemsList.length) {
  // subtract the space taken up by the mobile menu
  availableSpace -= this.mobileMenuContainer.offsetWidth;
}
```

then the comparison in the loop looks like `item.right <= availableSpace` again.